Src/PRuntimes/PSymRuntime/SymbolicRegressionTests/Integration/Correct/TwoPhaseCommit/TestDriver.p

```diff
@@ -10,7 +10,6 @@ machine Main {
             }
             coord = new Coordinator(participants);
 			new Client((coor = coord, n = 1));
```

Comment:
Are we not able to scale with 2 clients?

Reply:
These are regression tests for symbolic engine only in PSym CI. The idea is to finish them within few seconds.
With 2 clients, it requires more time and memory, unsuitable for regression testing.

Note that no changes are made to P/Tutorial folder in this PR.